fedmsg_migration_tools/bridges.py

```diff
@@ -41,126 +39,52 @@ def zmq_to_amqp(amqp_url, exchange, zmq_endpoints, topics):
         sub_socket.setsockopt(zmq.SUBSCRIBE, topic)
         _log.info('Configuring ZeroMQ subscription socket with the "%s" topic', topic)
 
-    retry_policy = {
-        'interval_start': 0,
-        'interval_step': 2,
-        'interval_max': 15,
-        'max_retries': 60,
-    }
-    with connections[Connection(amqp_url)].acquire(block=True, timeout=60) as conn:
-        exchange = Exchange(name=exchange, type='topic', durable=True)
-        producer = conn.Producer(auto_declare=True, exchange=exchange)
-        _log.info('Successfully connected to the AMQP broker; publishing to the "%s" '
-                  'exchange', exchange.name)
-        while True:
-            try:
-                topic, body = sub_socket.recv_multipart()
-            except zmq.ZMQError as e:
-                _log.error('Failed to receive message from subscription socket: %s', str(e))
-                continue
-            except ValueError as e:
-                _log.error('Unable to unpack message from pair socket: %s', e)
-                continue
-
-            _log.debug('Publishing %r to %r', body, topic)
-            try:
-                producer.publish(body, routing_key=topic, retry=True, retry_policy=retry_policy)
-            except Exception as e:
-                _log.exception('Publishing "%r" to exchange "%r" on topic "%r" failed (%r)',
-                               body, exchange, topic, e)
+    while True:
+        try:
+            topic, body = sub_socket.recv_multipart()
+        except zmq.ZMQError as e:
+            _log.error('Failed to receive message from subscription socket: %s', e)
+            continue
+        except ValueError as e:
+            _log.error('Unable to unpack message from pair socket: %s', e)
+            continue
+
+        message = fedora_messaging.message.Message(
```

Comment:
I guess we don't need to worry about picking a better schema here because the publisher is using fedmsg so they've probably not written one, right?